### DISCUSSION TITLE
Deepgram's speech to text not working on IOS

### DISCUSSION THREAD

**Discussion by Grzetan** [deepgram/community#729] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

Hello, I have a problem with Speech to text on IOS devices. 
When I talk only the first 3/4 words get processed, after that no other responses are received.
The same code works fine on Android, Firefox and Chrome.

audio/ogg; codecs=opus - firefox

audio/webm;codecs=opus - chrome/android

audio/mp4 - ios

So I guess there is a problem with mp4 format

Thanks in advance

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
